podkrepi-bg/frontend
# bugfix/1912-Support-Page-Forms-Validation

Implemented dynamic form validation for each step of the support form


Closes [#1912](https://github.com/podkrepi-bg/frontend/issues/1912)

## Motivation and context

On blur validation has been disabled due to an issue caused by the initial focus on the first input field of a form step

Logic is that each step will validated on change only when it was submitted with an error from the validation schema for that step. If a user goes back to a previous step and tries to submit it again with an error that logic will again be triggered









## Testing

### Steps to test

1. Navigate to https://podkrepi.bg/en/support
2. Try to submit each step without selecting/filling anything
3. Observe validation behavior for each step
4. Go back to previous steps and try to submit without selecting/filling anything and observe validation behavior

### Affected urls







## Environment

### New environment variables:




### New or updated dependencies:



Dependency name|Previous version|Updated version|Details
---|---|---|---
`dependency/name`|`v1.0.0`|`v2.0.0`|
